flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -504,7 +504,7 @@
 					"$(PROJECT_DIR)/Flutter/ios_backend_fw_static.framework",
 				);
 				MARKETING_VERSION = 0.7.1;
-				PRODUCT_BUNDLE_IDENTIFIER = com.xored.flutter1;
+				PRODUCT_BUNDLE_IDENTIFIER = org.mlcommons.flutter1;
```

Comment:
It's not like this matters too much for debug builds, but could you also remove this `1` from the end of the id?

Reply:
Same goes for profile build id.

Reply:
I leave the `flutter1` there because I don't know which Apple Team ID you are using. If you are not using the Team ID from MLCommons, the bundle ID will be reversed and MLCommons cannot use that bundle ID anymore.
The final bundle ID should be `org.mlcommons.mlperfbench` I think

Reply:
Ok, makes sense.

Reply:
Are you using the Team ID from MLCommons? Then I will change the bundle ID to the final one.

Reply:
I use my own account on my mac.
Locally I change the debug build bundle ID to my own unique ID with my name in it, so that it is guaranteed to not clash with anything.

Reply:
I only made this comment because `flutter1` looks kinda ugly. Not too much of an issue, just a personal preference.